js_modules/dagit/packages/core/src/gantt/GanttChart.tsx

```diff
@@ -387,7 +387,7 @@ const GanttChartInner = (props: GanttChartInnerProps) => {
           <GraphQueryInput
             items={props.graph}
             value={props.selection.query}
-            placeholder="Type a Step Subset"
+            placeholder="Type a step subset"
```

Comment:
Unrelated change, but we have this phrase uncapitalized as a placeholder elsewhere in the product